test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -242,6 +260,12 @@ generic_end_per_testcase(CaseName, Config) ->
 %% Service discovery test
 %%--------------------------------------------------------------------
 
+test_advertised_endpoints_override_endpoints(Config) ->
```

Comment:
Description on how this test works like.
What exactly we are comparing?

Example:

Requires module blabla to be started with argument advertised_endpoints for each host in get_hosts().
Reads Redis to confirm that endpoints are overwritten.

Reply:
@arcusfelis, in what form and where should such a description be put? What do you mean? (or do you mean - in a testcase name?)

Reply:
@kanes115 We can put some small readable description for each testcase function.
Example:

```erlang
%% This case verifies advertised_endpoints configuration parameter.
%% We read a list of endpoints from redis and compare it with configured value.
%% Makes this check for each host.
%% Requires module mod_global_distrib to be loaded with advertised_endpoints parameter.
test_advertised_endpoints_override_endpoints(Config) ->
...
```

Basically, think, that one of our clients asks:
- make me a document that describes each test you make, so we can ensure that our system works.
Than you can't just copy past them a list from `all()` function.

Reply:
I like that idea, but I don't see much documentation elsewhere in this suite. That's why I haven't documented it. No problem.

Reply:
During last grooming we've discussed that every new test case should have a brief description as a comment.

Reply:
@kanes115 We are going to have a meeting to discuss how docs and test cases should look like pretty soon :)
I often find myself in a situation, when I don't remember how my own tests work. But if there is some description, it's easier to dig into. Also, it helps to make a better test, the same motivation as duck programming.